include/mesh/dim3/coordinates/coordinates.hpp

```diff
@@ -108,6 +108,12 @@ template <> struct coordinates<specfem::dimension::type::dim3> {
    */
   std::string print(int ispec, specfem::mesh::mapping<dimension> &mapping,
                     const std::string component) const;
+
+  /**
+   * @brief Compute bounding box of the mesh`
+   *
+   */
```

Comment:
```suggestion
  /**
   * @brief Compute bounding box of the mesh`
   *
   * @return std::array<type_real, 6> Array containing minima and maxima of domain @code {xmin, xmax, ymin, ymax, zmin, zmax} @endcode
   */
```

Just added this since I was confused on what this returned until I saw the `.cpp` file